packages/progress-bar/src/ProgressBar.ts

```diff
@@ -146,6 +146,15 @@ export class ProgressBar extends SizedMixin(
         }
 
         if (window.__swc.DEBUG) {
+            if (changes.has('over-background')) {
+                window.__swc.warn(
+                    this,
+                    `The "over-background" attribute on <${this.localName}> has been deprecated and will be removed in a future release. Use "static-color='white'" instead.`,
+                    'https://opensource.adobe.com/spectrum-web-components/components/progress-bar/#variants',
+                    { level: 'deprecation' }
+                );
+            }
```

Comment:
can you write a test for this?

Reply:
Can you check below some lint failures in the test suite is blocking the pipeline to complete.
https://app.circleci.com/pipelines/github/adobe/spectrum-web-components/31826/workflows/6ea58690-5783-4110-8f94-4219730d7bd5/jobs/827391